articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,147 @@
+---
+section: libraries
+toc: true
+description: Migration Guide for lock-passwordless to Lock 10 with Passwordless Mode
+---
+# Migration Guide for lock-passwordless to Lock 10 with Passwordless Mode
+
+The following instructions assume you are migrating from **lock-passwordless** to the latest **Lock 10** using the **Passwordless** mode.
+
+The [lock-passwordless](https://github.com/auth0/lock-passwordless) widget was previously a standalone library, separate from [Lock 10](https://auth0.com/docs/libraries/lock/v10). Now, you can migrate your apps to use the newest Lock Passwordless, which is integrated directly into Lock 10. Lock 10 with Passwordless Mode is the latest method by which to quickly and simply deploy a login widget for passwordless authentication in your apps.
```

Comment:
`[Lock 10](https://auth0.com/docs/libraries/lock/v10)` should be relative ie `[Lock 10](/libraries/lock/v10)`